openedx-unsupported/devstack
# Target changed to dev image for devstack for LMS & Studio

**Task description:**
https://github.com/openedx/edx-platform/issues/31737

**How to test**
- Go to devstack
- Make sure you have latest edx-platform repo in workspace
- Now, all set to up LMS
- Run `make dev.up.lms`

**Commands to test**
- Make sure `make dev.shell.lms` is running fine and opening shell
- Make sure `make dev.provision.lms` is running fine
- Make sure `make dev.logs.lms` is not throwing any errors, it should run fine and showing logs
- In case, you see issue on [dashboard](http://localhost:18000/dashboard), just open lms shell and run migrations and re-try
- Migration command --> `python manage.py lms migrate`
